x-pack/test/security_solution_cypress/cypress/tasks/asset_criticality/common.ts

```diff
@@ -26,6 +28,7 @@ export const expandFirstAlertHostFlyout = () => {
  * Find the first alert row in the alerts table then click on the host name to open the flyout
  */
 export const expandFirstAlertUserFlyout = () => {
+  scrollAlertTableColumnIntoView(USER_COLUMN);
```

Comment:
It didn't click when the column was not visible.

Reply:
AHHHH! So this must be why the tests I disabled were also flaky